tools/export/blender25/godot_export_manager.py

```diff
@@ -3,38 +3,42 @@
 # This program is free software; you can redistribute it and/or
 # modify it under the terms of the GNU General Public License
 # as published by the Free Software Foundation; either version 2
-# of the License, or (at your option) any later version.
+# of the License,  or (at your option) any later version.
 #
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 # GNU General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
-# along with this program; if not, write to the Free Software Foundation,
-# Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+# along with this program; if not,  write to the Free Software Foundation,
+# Inc.,  51 Franklin Street,  Fifth Floor,  Boston,  MA 02110-1301,  USA.
 #
 # ##### END GPL LICENSE BLOCK #####
 
 # Script copyright (c) Andreas Esau
 
+import bpy
+from bpy.props import (StringProperty,  BoolProperty,  EnumProperty,
+                       FloatProperty,  IntProperty, CollectionProperty)
+import os
+from bpy.app.handlers import persistent
+from mathutils import Matrix
+
 bl_info = {
     "name": "Godot Export Manager",
     "author": "Andreas Esau",
-    "version": (1, 0),
-    "blender": (2, 7, 0),
+    "version": (1,  0),
```

Comment:
Bad change too, just running `sed -e 's/,/, /g'` is a bit too drastic if you don't check the diff afterwards...